### PR TITLE
Installing Visio/Project volume license on top of Office365PP existin…

### PR DESCRIPTION
**Pull request by Blasteyes**
…g install

The caveat is very confusing, you can install Visio perpetual on top of O365PP but you have to use different channel when you do setup.exe /download and change the channel to Monthly/Broad depending on O365PP install when you do setup.exe /configure

### CODE REVIEW

File: DeployOffice/install-different-office-visio-and-project-versions-on-the-same-computer.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ Here are some example installation scenarios, whether they’re supported, and a
...
-- For Office 365 and Office 2019 products, all products installed on the computer must be using the same [update channel](overview-of-update-channels-for-office-365-proplus.md). For example, the volume licensed version of Office Professional Plus 2019 can only use the PerpetualVL2019 update channel. Therefore, it can’t be installed on the same computer that has Project Online Desktop Client using Semi-Annual Channel or on the same computer that has Visio Online Plan 2 (the subscription version of Visio that was previously named Visio Pro for Office 365) using Monthly Channel.
+- For Office 365 and Office 2019 products, all products installed on the computer must be using the same [update channel](overview-of-update-channels-for-office-365-proplus.md). For example, the volume licensed version of Office Professional Plus 2019 can only use the PerpetualVL2019 update channel. Therefore, download the package using PerpetualVL2019 mentioned in channel and change it to channel of Office365ProPlus when you configure.
```
Comment (ItzLevvie) [inline DeployOffice/install-different-office-visio-and-project-versions-on-the-same-computer.md]:
```suggestion
- For Office 365 and Office 2019 products, all products installed on the computer must be using the same [update channel](overview-of-update-channels-for-office-365-proplus.md). For example, the volume licensed version of Office Professional Plus 2019 can only use the PerpetualVL2019 update channel. Therefore, download the package using PerpetualVL2019 mentioned in channel and change it to channel of Office 365 ProPlus when you configure. 
```

Comment (ItzLevvie) [inline DeployOffice/install-different-office-visio-and-project-versions-on-the-same-computer.md]:
```suggestion
- For Office 365 and Office 2019 products, all products installed on the computer must be using the same [update channel](overview-of-update-channels-for-office-365-proplus.md). For example, the volume licensed version of Office Professional Plus 2019 can only use the PerpetualVL2019 update channel. Therefore, download the product using the PerpetualVL2019 update channel and change it the Monthly update channel when you configure.
```

**Comment by Aliencompit12** [comment]
Blasteyes:patch-1

**Review by Aliencompit12 — COMMENTED** [review]
Blasteyes:patch-1

**Review by Aliencompit12 — COMMENTED** [review]
gh pr checkout 408

**Comment by DHB-MSFT** [comment]
Volume licensed (aka "perpetual") versions of Project 2019 and Visio 2019 support all the same channels that Microsoft 365 Apps (previously named Office 365 ProPlus) do. 

So if you are doing /download, you should set the Channel attribute in the Add element to the same channel as the Microsoft 365 Apps install on the device. 

If you are installing directly from the internet (/configure), don't include the Channel attribute. Install is smart enough to see what channel Microsoft 365 Apps is using and have Visio or Project install use that channel. 

Although channels aren't that significant for perpetual products, because they don't get any new feature updates after they ship (which is what channels are used for in Microsoft 365 Apps).

**Pull request closed without merging**